source/blog/2016-11-11-testing-ovirt-changes-without-cluster.md

```diff
@@ -0,0 +1,188 @@
+
+```
+# "${PREFIX}/bin/engine-setup"
+```
+
+
+
+The last step that needs to be done is related to the fake hosts we will be using. The host simulator does not support encrypted communication so we need to disable it:
+
+```
+sudo -i -u postgres psql $ENGINE_DB -c "UPDATE vdc_options set option_value = 'false' WHERE option_name = 'SSLEnabled';"
+sudo -i -u postgres psql $ENGINE_DB -c "UPDATE vdc_options set option_value = 'false' WHERE option_name = 'EncryptHostCommunication';"
+```
+
+The fake host is no real host so we also have to disable host deploy (the engine would otherwise try to update and configure the development machine when provided by proper credentials!):
+
+```
+psql $ENGINE_DB -c "UPDATE vdc_options set option_value = 'false' where option_name = 'InstallVds'"
+```
+
+And the last option instructs the engine to send the hostname with every request that goes to the host. This allows us to use a single vdsmfake instance to simulate multiple hosts at once:
+
+```
+psql $ENGINE_DB -c "UPDATE vdc_options set option_value = 'true' WHERE option_name = 'UseHostNameIdentifier';"
+```
+
+Starting up the engine and adding the fake hosts
+A properly deployed and configured engine can be now started using the following command:
+
+```
+$ "${PREFIX}/share/ovirt-engine/services/ovirt-engine/ovirt-engine.py" start
+```
+
+Once the service boots up, you can access it by going to [http://localhost:8080](http://localhost:8080) and logging in using the engine credentials you provided during the setup phase.
+
+
+
+![oVirt welcome screen](/images/vdsmfake/welcome.png)
+
+
+
+Our fake hosts can be started by first cloning the [source](git://gerrit.ovirt.org/ovirt-vdsmfake.git), entering the ovirt-vdsmfake directory and using maven to start the [jetty](http://www.eclipse.org/jetty/) container:
+
+```
+# git clone git://gerrit.ovirt.org/ovirt-vdsmfake.git
+# pushd ovirt-vdsmfake
+# mvn clean jetty:run
+```
+
+
+
+You can now add as many hosts as you need to the engine using any made up names and addresses (just suffix them with `.vdsm.fake`). The engine requires you to enter a password when adding a host, but any random string is fine there (it won't be used anyway).
+
+
+![adding a new host](/images/vdsmfake/new-host.png)
+
+
+
+You have to connect a storage domain to make the engine fully operational so go to the Storage tab and create a new storage domain using any random NFS path. The fake hosts will report a connection success without actually talking to any remote server and this will enable the Datacenter functionality.
+
+![adding a new storage domain](/images/vdsmfake/storage.png)
+
+
+## What to do next?
+
+
+The setup is now ready and you can use most of the functionality to test your changes or just play with the engine. Starting, stopping and migrating VMs will work (no real VM will exist though, all is faked by the host simulator) as will most of the other administrative flows.
```

Comment:
Add a comma after "stopping".

My suggestion: "(no real VMs will exist though - everything is faked by the host simulator)"